cmd/bosun/sched/sched.go

```diff
@@ -259,11 +282,16 @@ type StateGroups struct {
 }
 
 func (s *Schedule) MarshalGroups(T miniprofiler.Timer, filter string) (*StateGroups, error) {
-	silenced := s.Silenced()
```

Comment:
Not strictly speaking. Removing timelock made sense since every lock is timelock now. Adding a step around silence was useful because I suspect that is where most of the time is spent. This whole thing is really to diagnose why this endpoint is getting locked.